lib/ProgressPlugin.js

```diff
@@ -348,10 +349,16 @@ class ProgressPlugin {
 					}
 				);
 			}
+			invalidatedCache = false;
+		});
+
+		compiler.cache.hooks.store.tap("ProgressPlugin", () => {
+			invalidatedCache = true;
 		});
 
 		compiler.hooks.afterCompile.tapPromise("ProgressPlugin", compilation => {
-			if (compilation.compiler.isChild()) return Promise.resolve();
+			if (compilation.compiler.isChild() || !invalidatedCache)
```

Comment:
this will not cache at all.. we can check `oldData` on line 364, but when update this info?